### PR TITLE
Windows/v6

### PR DESCRIPTION
**Pull request by victorjulien**
Replaces #13. Now defaults to --disable-detection for the case where a test has no rules.

### CODE REVIEW

**Comment by victorjulien** [comment]
@shivan1b can you review the changes?

**Comment by shivan1b** [comment]
Following the rest of the changes, this should also have the Windows check probably: https://github.com/OISF/suricata-verify/blob/6ece72b925b53ba9623525129e7fb15527aad644/run.py#L111

File: run.py [hunk 1]
```diff
@@ -167,14 +167,20 @@ def __init__(self, version):
         self.load_build_info()
 
     def load_build_info(self):
-        output = subprocess.check_output(["./src/suricata", "--build-info"])
+        cmd = "./src/suricata"
+        if sys.platform == "win32":
+            cmd = "src\suricata.exe"
```
[the remark this answers is not in the document]
Comment (victorjulien) [reply run.py]: Trying this indeed.

File: run.py [hunk 2]
```diff
@@ -644,13 +661,20 @@ def start_reader(self, input, output):
 
 def check_deps():
     try:
-        subprocess.check_call("jq --version > /dev/null 2>&1", shell=True)
+        cmd = "jq --version > /dev/null 2>&1"
+        if sys.platform == "win32":
```
[the remark this answers is not in the document]
Comment (victorjulien) [reply run.py]: Doing this, although I wonder how do this and avoid the line becoming too long.

File: run.py [hunk 3]
```diff
@@ -687,11 +711,18 @@ def main():
...
+                os.path.exists("src\suricata.exe")):
+            print("error: this is not a suricata source directory or " +
+                  "suricata is not built")
+            return 1
+    else:
+        if not (os.path.exists("./suricata.yaml") and
+                os.path.exists("./src/suricata")):
+            print("error: this is not a suricata source directory or " +
+                  "suricata is not built")
+            return 1
```
[the remark this answers is not in the document]
Comment (victorjulien) [reply run.py]: Going with the global locations suggested below.

**Comment by victorjulien** [comment]
Updating the get_suricata_version as well

**Comment by victorjulien** [comment]
Replaced by #37

**Pull request closed without merging**